app/Http/Controllers/PaymentRequestController.php

```
@@ -82,11 +84,17 @@ public function myDeniedRequests(PaymentRequestFilter $request)
 
     public function searchStakeHolders(StakeholderRequestFilter $request)
     {
-        return new JsonResponse([
+        $validatedData = $request->validated();
+        $query = StakeHolder::where('name', 'like', '%'.strtolower($validatedData['key']).'%');
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\StakeHolder' in method 'searchStakeHolders'. (undefined)

(StaticAccess)